lighthouse-cli/test/smokehouse/test-definitions/core-tests.js

```diff
@@ -5,71 +5,217 @@
  */
 'use strict';
 
-/** @type {ReadonlyArray<Smokehouse.BackCompatTestDefn>} */
+const errors = require('./errors/error-expectations.js');
+const pwa = require('./pwa/pwa-expectations.js');
+const pwa2 = require('./pwa/pwa2-expectations.js');
+const redirects = require('./redirects/expectations.js');
+const seo = require('./seo/expectations.js');
+const offline = require('./offline-local/offline-expectations.js');
+const byte = require('./byte-efficiency/expectations.js');
+const perf = require('./perf/expectations.js');
+const diagnostics = require('./perf-diagnostics/expectations.js');
+const lantern = require('./lantern/lantern-expectations.js');
+const metrics = require('./tricky-metrics/expectations.js');
+const csp = require('./csp/csp-expectations.js');
+
+/** @type {ReadonlyArray<Smokehouse.TestDfn>} */
 const smokeTests = [{
   id: 'a11y',
   expectations: require('./a11y/expectations.js'),
   config: require('./a11y/a11y-config.js'),
 }, {
-  id: 'errors',
-  expectations: require('./errors/error-expectations.js'),
+  id: 'errors-infinite-loop',
+  expectations: errors.infiniteLoop,
+  config: require('./errors/error-config.js'),
+  runSerially: true,
+}, {
+  id: 'errors-expired-ssl',
+  expectations: errors.expiredSsl,
+  config: require('./errors/error-config.js'),
+  runSerially: true,
+}, {
+  id: 'errors-iframe-expired-ssl',
+  expectations: errors.iframeBadSsl,
   config: require('./errors/error-config.js'),
   runSerially: true,
 }, {
   id: 'oopif',
   expectations: require('./oopif/oopif-expectations.js'),
   config: require('./oopif/oopif-config.js'),
 }, {
-  id: 'pwa',
-  expectations: require('./pwa/pwa-expectations.js'),
+  id: 'pwa-airhorner',
+  expectations: pwa.airhorner,
   config: require('./pwa/pwa-config.js'),
 }, {
-  id: 'pwa2',
-  expectations: require('./pwa/pwa2-expectations.js'),
+  id: 'pwa-chromestatus',
+  expectations: pwa.chromestatus,
   config: require('./pwa/pwa-config.js'),
 }, {
-  id: 'pwa3',
-  expectations: require('./pwa/pwa3-expectations.js'),
+  id: 'pwa-svgomg',
+  expectations: pwa2.svgomg,
+  config: require('./pwa/pwa-config.js'),
+}, {
+  id: 'pwa-caltrain',
+  expectations: pwa2.caltrain,
+  config: require('./pwa/pwa-config.js'),
+}, {
+  id: 'pwa-rocks',
+  expectations: require('./pwa/pwa3-expectations.js').pwarocks,
   config: require('./pwa/pwa-config.js'),
 }, {
   id: 'dbw',
   expectations: require('./dobetterweb/dbw-expectations.js'),
   config: require('./dobetterweb/dbw-config.js'),
   runSerially: true, // Need access to network request assertions.
 }, {
-  id: 'redirects',
-  expectations: require('./redirects/expectations.js'),
+  id: 'issues-mixed-content',
+  expectations: require('./issues/mixed-content.js'),
+}, {
+  id: 'redirects-single-server',
```

Comment:
how about going one step further and moving this entire object to what `expectations.js` exports? then this big chunk of code is just bringing it all together as an array.

Reply:
Yeah, it's a little silly to have filenames for separate files and then IDs for them in a different file. That'll be in the above-mentioned (but undescribed, sorry) next PR.